earth_enterprise/src/common/khCache.h

```diff
@@ -162,24 +171,59 @@ class khCache {
  public:
   typedef typename MapType::size_type size_type;
   size_type size(void) const { return map.size(); }
-  size_type capacity(void) const { return targetMax; }
+  uint64 capacity(void) const { return targetMax; }
+  uint64 getMemoryUse(void) const { return cacheObjectSizes; }
+  uint64 itemSizeMinusKeyAndValue(void) const { return sizeof(khCacheItem<Key, Value>) - sizeof(Key) - sizeof(Value); }
```

Comment:
Minor: this could be a constant instead of a function, which might be a bit faster.